Docs/XCHammerFAQ.md

```diff
@@ -0,0 +1,39 @@
+## XCHammer Frequently Asked Questions
+
+## Why XCHammer?
+
+Many features in Xcode rely on the fact that Xcode is both the editor and build
+system. Since XCHammer builds Bazel projects both Xcode and Bazel as a core
+design goal, it supports all of these features by default.
+
+_XCHammer generates an Xcode project from Bazel rules, which we've found to be a
+great way to describe an Xcode project and corresponding apps!_
+
+## How is XCHammer versioned?
+
+XCHammer releases correspond to a tested, canonical, Bazel release.
+
+For Apple based development, XCHammer depends on attributes of
+[`rules_apple`](https://github.com/bazelbuild/rules_apple). The current, tested,
+version is updated in the [sample
+WORKSPACE](https://github.com/pinterest/xchammer/blob/master/sample/UrlGet/WORKSPACE).
+
+## How can I develop XCHammer?
+
+The `Makefile` contains commands for development and deployment. `make run` and `make run_force` both run the sample app with a debug build.
+
+*The sample is a great way to do development on, and the authors recommend adding examples of edge cases and bugs here :). Pull requests welcome!*
+
+## How can I develop XCHammer with Xcode?
+
+To generate an Xcode project, use the make command, `make workspace`. Make sure, that in the scheme, to setup absolute paths to the `WORKSPACE` and `XCHammerConfig`.
```

Comment:
Can you test this with a fresh clone? Last time I checked I needed to run `swift package generate-xcodeproj` before `make workspace` since that depends on `make aspects` which requires some of the workspace to be setup first

Reply:
@rahul-malik it should work - I think there was a bug back in the day that caused this.